src/application/utils/tokenForAuth/generateToken.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+import * as jwt from 'jsonwebtoken';
+import { IUser } from '../../../interfaces/user.interface';
+
+dotenv.config();
+
+export const secretKey = process.env.SECRET_KEY as string;
+export const refreshTokenSecretKey = process.env.REFRESH_SECRET_KEY as string;
+
+
+export function generateAccessToken(user: IUser): string {
+  const token = jwt.sign(
+    { userId: user.id, username: user.username },
+    secretKey,
+    { expiresIn: '1h' },
```

Comment:
it would better to move as many properties as we can to `.env` or any other property source, then we can run our app in more flexible way
so I think it would better to store expiration period like this

Reply:
Yes, good idea) I will store expiresIn for refresh and access tokens in .env